ts/Extensions/Drilldown.ts

```diff
@@ -1119,7 +1121,15 @@ ColumnSeries.prototype.animateDrillupTo = function (init?: boolean): void {
         // Do dummy animation on first point to get to complete
         syncTimeout(function (): void {
             if (newSeries.points) { // May be destroyed in the meantime, #3389
-                newSeries.points.forEach(function (
+                // Unable to drillup with nodes, #13711
+                var seriesWithNodes: Array<any> = [];
```

Comment:
The name `seriesWithNodes` is a bit confusing, because this array actually contains Point items. So I suggest `pointsWithNodes` is a better name.